index.html

```diff
@@ -1,11 +1,40 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <link rel="stylesheet" href="index.css" />
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Bio Page</title>
 </head>
-<body>
-    
+<body bgcolor="beige">
+    <div align="center">
+        <img id="img" src="image/anu.jpeg">
+        <br>
+        <h2 style="text-align:center;line-height: 5px;">Anuradha Sathiyachalam</h2>
+        <h3 style="text-align:center;line-height: 5px;">Dallas, TX</h3>
+        <h3 style="text-align:center;line-height: 5px;">Email: anoosathyee@gmail.com</h3>
```

Comment:
Avoid using the 'style' attribute in your HTML, and instead apply styles from your style sheet (index.css).